### PR TITLE
fix: Added gas deviation to the release-light tests.

### LINKED ISSUE #3259: The `release-light` test suite needs to account for gas deviation.
### Description

The `release-light` test suite is used to run a small set of tests post deployment to mainnet.  In the last runs on previewnet and testnet there were errors like:
```
Error: insufficient funds for intrinsic transaction cost (transaction="0x02f874820129048086014a9e9878008308f4f8940764cd5ff373bcc89f5ba73112b2b89fd6c7cac78902b5e3af16b188000080c080a0a64a00afb1386acd3fe179cbf3a085a98d3980e15d06ee70ec99a120e9c4bcdfa0202f94cad9b00ffe232eeb9c83e74d0eddc3f33dab911f20c4ce44c000d7d7ee", info={ "error": { "code": -32000, "message": "[Request ID: e0cb0d2f-c903-4a8c-ac01-000ffda05b91] Insufficient funds for transfer" } }, code=INSUFFICIENT_FUNDS, version=6.13.2)



@release-light, @release should execute "eth_sendRawTransaction" for legacy EIP 155 transactions:
     Error: server response 400 Bad Request (request={  }, response={  }, error=null, info={ "requestUrl": "https://testnet.hashio.io/api", "responseBody": "{\"error\":{\"code\":-32009,\"message\":\"[Request ID: b57bf3d0-9b54-45a3-8563-45851d92b402] Gas price '1380000000000' is below configured minimum gas price '1420000000000'\"},\"jsonrpc\":\"2.0\",\"id\":18}", "responseStatus": "400 Bad Request" }, code=SERVER_ERROR, version=6.13.2)
 ```

### Steps to reproduce

Run the `Acceptance Tests on Public` workflow during business hours.  It does not always fail.  

### Additional context

_No response_

### Hedera network

testnet

### Version

v0.59.4

### Operating system

Linux

### PR DESCRIPTION
**Pull request by ebadiere**
This PR adds a gas deviation to the gas price in the `@release-light` tests, and removes some warnings in the IDE.

**Related issue(s)**:

Fixes #3259 

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: packages/server/tests/acceptance/rpc_batch1.spec.ts [hunk 1]
```diff
@@ -795,11 +794,13 @@ describe('@api-batch-1 RPC Server Acceptance Tests', function () {
       });
 
       it('@release-light, @release should execute "eth_getTransactionReceipt" for hash of legacy transaction', async function () {
+        const gasPrice = await relay.gasPrice(requestDetails);
+        const gasPriceWithDeviation = gasPrice * (1 + gasPriceDeviation);
```
Comment (Nana-EC) [inline packages/server/tests/acceptance/rpc_batch1.spec.ts:798]: Move this to a shared method

File: packages/server/tests/acceptance/rpc_batch1.spec.ts [hunk 2]
```diff
@@ -68,17 +68,18 @@ describe('@api-batch-1 RPC Server Acceptance Tests', function () {
   let account2Address: string;
   let expectedGasPrice: string;
 
-  const CHAIN_ID = ConfigService.get('CHAIN_ID') || '0x12a';
+  const CHAIN_ID = (ConfigService.get('CHAIN_ID') as string) || '0x12a';
```
Comment (Nana-EC) [inline packages/server/tests/acceptance/rpc_batch1.spec.ts:71]: is there a bug that's being fixed here?
It seems right just curious if it's impacting any tests

File: packages/server/tests/acceptance/rpc_batch1.spec.ts [hunk 3]
```diff
@@ -68,17 +68,18 @@ describe('@api-batch-1 RPC Server Acceptance Tests', function () {
...
   const requestId = 'rpc_batch1Test';
   const requestIdPrefix = Utils.formatRequestIdMessage(requestId);
-  const requestDetails = new RequestDetails({ requestId: 'rpc_batch1Test', ipAddress: '0.0.0.0' });
+  const requestDetails = JSON.stringify(new RequestDetails({ requestId: 'rpc_batch1Test', ipAddress: '0.0.0.0' }));
```
Comment (Nana-EC) [inline packages/server/tests/acceptance/rpc_batch1.spec.ts:74]: Same question, if there a bug being fixed here?

**Review by Nana-EC — CHANGES_REQUESTED** [review]
Some improvement suggestions.
Also please update the description, there's some changes that does seem related and might relate to other issues that need to be highlighted.

File: packages/server/tests/acceptance/rpc_batch1.spec.ts (hunk 2 above)
Comment (ebadiere) [reply packages/server/tests/acceptance/rpc_batch1.spec.ts:71]: This just removes a warning in the IDE.

File: packages/server/tests/acceptance/rpc_batch1.spec.ts (hunk 3 above)
Comment (ebadiere) [reply packages/server/tests/acceptance/rpc_batch1.spec.ts:74]: This just removed another warning in the IDE.

File: packages/server/tests/acceptance/rpc_batch1.spec.ts (hunk 2 above)
Comment (ebadiere) [reply packages/server/tests/acceptance/rpc_batch1.spec.ts:71]: Done.

Comment (quiet-node) [reply packages/server/tests/acceptance/rpc_batch1.spec.ts:71]: Just FYI these are known issues and capture in this ticket https://github.com/hashgraph/hedera-json-rpc-relay/issues/3142

**Review by quiet-node — APPROVED** [review]
LGTM

**Review by Nana-EC — APPROVED** [review]

**Pull request merged**